### ISSUE TITLE
Is there anyone trained deformable-detr on voc2007? 

### ISSUE THREAD

**Issue by order-a-lemonade** [fundamentalvision/Deformable-DETR#189]
hello, i trained deformable-detr on voc2007 recently, however the performence is not so good.  my setting is: epoch=150, initLR=2e-4, lr-drop at 55epoch and 110 epoch, batchsize=1node*6gpus*2,  the results are as follows:

![image](https://user-images.githubusercontent.com/66902842/223333149-3e71337e-47d4-4fbd-83e4-36c0c58c4fcb.png)
the ap50 is 71.6, and i think it's too low for deformable-detr. i had tried to use the weights which is trained on coco to evaluate on voc2007 testset, and i got 78.4 ap50.

and my loss curves are as follows:
![image](https://user-images.githubusercontent.com/66902842/223333587-c7427e9a-d3b5-48fd-9e0f-4bf135d20199.png)
it seems like the model overfits on train dataset very early. 
Is there any suggestions? or is anyone got an reasonable results on voc2007? 
really appreciate you guys!
